event/src/event.rs

```diff
@@ -0,0 +1,138 @@
+use crate::args::UnsignedEvent;
+use crate::DidDocument;
+use anyhow::Result;
+use ceramic_core::{Cid, DagCborEncoded, Jwk, Jws};
+use multihash::{Code, MultihashDigest};
+use serde::Serialize;
+
+// https://github.com/multiformats/multicodec/blob/master/table.csv
+const DAG_CBOR_CODEC: u64 = 0x71;
+
+/// A ceramic event, which is a signed commit
+pub struct Event {
+    /// Cid of the data for the event
+    pub cid: Cid,
+    /// The data for the event to be encoded in the block
+    pub linked_block: DagCborEncoded,
+    /// JWS signature of the event
+    pub jws: Jws,
+}
```

Comment:
Probably don't need to change this now, but events can also contain one `cacaoBlock` (currently), and possible multiple CACAO blocks in the future.

I wonder if a better structure for an event is just a CAR file with all the relevant IPLD blocks?

If not clear already: both `jws` and `linked_block` should be IPLD objects.